queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
+-- SELECT * from animals WHERE name = 'Luna';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
```

Comment:
- [ ] Let's filter all animals' names and find the animals that have "mon" at the end. Currently, you are filtering only names but it's highly recommended if we filter everything such as: 
`SELECT * FROM animals WHERE name LIKE '%mon';`

Reply:
Done